### ISSUE TITLE
Allow fatalError without message in unavailable methods

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#1379]
It's a common pattern to add `fatalError` without a message in methods marked as unavailable.

For example:
```swift
@available(*, unavailable, renamed: "dateComponents(_:from:to:)")
public func components(_ unitFlags: NSCalendar.Unit, from startingDate: Date, to resultDate: Date, options opts: NSCalendar.Options = []) -> DateComponents {
    fatalError() 
}
```

In this case, we shouldn't trigger a violation. This would enable us to make the `fatal_error_message` rule enabled by default.
